src/fdc3/js/composeui-fdc3/package.json

```diff
@@ -23,6 +23,14 @@
     "jest-environment-jsdom": "^29.7.0",
     "rxjs": "^7.8.1"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/morganstanley/ComposeUI.git"
+  },
+  "publishConfig": {
+    "access": "public",
```

Comment:
if we don't want to release the composeui-fdc3 package now this should be private. The access is for npm publish, or for any tool that wraps that (in our case Lerna)